app/src/main/res/drawable/ic_share_black_24dp.xml

```diff
@@ -4,6 +4,6 @@
         android:viewportWidth="24.0"
         android:viewportHeight="24.0">
     <path
-        android:fillColor="#FF000000"
+        android:fillColor="@color/black"
```

Comment:
Based on material design recommendation, it would be much better if we use`?attr/colorControlNormal` for icons' **fillColor**.
Take a look in [here](https://developer.android.com/guide/topics/ui/look-and-feel/darktheme), please

Reply:
I did see the standard style but since the color is in disabled state it doesn't look good.